tests/test_req_vars.py

```diff
@@ -647,3 +647,68 @@ def _test_error_details(self, headers, attr_name,
         except error_type as ex:
             self.assertEqual(ex.title, title)
             self.assertEqual(ex.description, description)
+
```

Comment:
It would be cool if you could also add a few HTTP/1.1 tests for the new properties (to ensure those code paths return what we expect).